bottles/frontend/ui/preferences.blp

```diff
@@ -126,32 +126,33 @@ template PreferencesWindow : .AdwPreferencesWindow {
 
     .AdwPreferencesGroup {
       title: _("Advanced");
-
       .AdwActionRow action_bottles_path {
-        title: _("Custom Bottles Path (Requires Restart)");
-        subtitle: _("Choose where to store the new bottles (this will not move the existing ones).");
-        activatable-widget: "btn_bottles_path";
-
-        Button btn_bottles_path_reset {
-          tooltip-text: _("Reset to default");
-          valign: center;
-          visible: false;
-          icon-name: "edit-undo-symbolic";
-
-          styles [
-            "flat",
-          ]
-        }
-
-        Button btn_bottles_path {
-          tooltip-text: _("Choose a directory");
-          halign: center;
-          valign: center;
-          icon-name: "document-open-symbolic";
-
-          styles [
-            "flat",
-          ]
+        title: _("Bottles Directory");
+        subtitle: _("Directory that contains the data of your Bottles.");
+        activatable: True;
+        .GtkBox {
+          spacing: 6;
+          Button btn_bottles_path_reset {
+            tooltip-text: _("Reset to default");
+            valign: center;
+            visible: false;
+            icon-name: "edit-undo-symbolic";
+            styles ["flat"]
+          }
+          .GtkImage {
+            icon-name: folder-symbolic;
+            styles ["dim-label"]
+          }
+          .GtkLabel bottles_path_label {
+            ellipsize: end;
+            xalign: 1;
```

Comment:
What is this property for? It doesn't change anything when I remove it locally

Reply:
epiphany used it so i also used these properties

Reply:
id say keep ellipsize as people might have some long folder names and we dont want the UI to overflow

Reply:
I didn't talk about the ellipsize. 
The xalign just changes the vertical alignment of the text (in this case up by 1), which isn't really needed.

Reply:
ok ill remove that

Reply:
ok i removed it. tell me if you want me to do anything else with the pr.